MovingBlocks/TerasologyLauncher
# 4K background image

### Contains
- New, 4K background image
- removed size constants of the image, so it is not scaled or fitted to a smaller view
Fixes #399

### How to test

Start the launcher, try to resize the launcher window, the image should be able to take full window on the 4K resolution.

### Outstanding before merging

- [x] Confirmation that background image takes up whole launcher window when maximised from someone with the 4K capable machine (I don't have access to one...)
